waflib/Node.py

```diff
@@ -771,19 +771,24 @@ def get_bld(self):
 		# <bld_dir>/__root__/abspath/to/file to avoid contamination of
 		# filesystem.
 		schrodinger_path = os.path.realpath(os.environ['SCHRODINGER'])
+		# $SCHRODINGER_SRC
 		schrodinger_src_path = os.path.realpath(os.environ['SCHRODINGER_SRC'])
+		# $SCHRODINGER_SRC/mmshare
```

Comment:
This could be other repo source directories as well, yes? Maybe just prefix the comment with "Eg: "